src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -192,6 +192,12 @@ Copyright (c) .NET Foundation. All rights reserved.
         <RelativePath>@(_DebugSymbolsIntermediatePath->'%(Filename)%(Extension)')</RelativePath>
       </ResolvedFileToPublish>
 
+      <!-- Copy the documentation file (.xml), if any -->
+      <ResolvedFileToPublish Include="@(_DebugSymbolsIntermediatePath)"
```

Comment:
I pushed up a change to include doc files in the TestLibrary.csproj build output. This actually does cause the test to fail because the files do get published and (according to the way the test is currently defined), these files are not expected. The thing is, this change to TestLibrary.csproj will cause this test to fail even in the absence of the change I made to the targets file.